motr/io_pargrp.c

```diff
@@ -297,7 +297,9 @@ static void data_buf_dealloc_fini(struct data_buf *buf)
 		m0_free_aligned(buf->db_buf.b_addr,
 				buf->db_buf.b_nob,
 				M0_NETBUF_SHIFT);
+	}
 
+	if (buf->db_auxbuf.b_addr != NULL) {
```

Comment:
It seems free function does not make this b_addr NULL, hope this does not create any issue?